phpBB/styles/prosilver/theme/common.css

```diff
@@ -1269,6 +1269,10 @@ ul.linklist:after,
 	padding: 4px 6px;
 }
 
+.badge.hidden {
+	display: none;
+}
+
```

Comment:
it should be important ideally but the issue is where it falls in the layering of rules, all important rules should be the last loaded in a utilities or trumps layer to solve inheritance issues. The main problem with phpbb hidden class is that the class is spiderweb'd in weird uses throughout the theme and the js improperly so we can not use the hidden class with important without causing major issues. for now its fine as is all though not optimal until we relocate the utilities layer to the end of the stylesheets.css file there is no way around it